modules/ztp-site-planning-for-du-deployments.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-site-planning-for-du-deployments_{context}"]
+= Site planning considerations for distributed unit deployments
+
+Site planning for distributed units (DU) deployments is complex. The following is an overview of the tasks that you complete before the DU hosts are brought online in the production environment.
+
+* Develop a network model. The network model will depend on various factors such as the size of the area of coverage, number of hosts, projected traffic load, network slicing, DNS and DHCP requirements, and so on.
```

Comment:
Double-check that future tense is required here.

Also, please replace "and so on"
> ! and so on noun
Use only when you list a clear sequence of elements, such as "1, 2, 3, and so on" or "Monday, Tuesday, Wednesday, and so on". When you list items that do not form a clear sequence, use appropriate descriptive wording, such as "device drivers, firmware, and other code".